editor/vscode/package.json

```
@@ -88,6 +89,7 @@
 	"scripts": {
 		"vscode:prepublish": "npm run compile",
```

Maybe this helps with the CI:

```suggestion
		"vscode:prepublish": "npm run compile && npm run compile-web",
```